Library/Formula/iojs.rb

```diff
@@ -1,4 +1,8 @@
+require "language/javascript"
+
 class Iojs < Formula
+  include Language::JS
```

Comment:
I'd be happy to go back and update it to decouple from the `Formula` API in the future once the correct ingredients are in place.

Leaving it as a mixin for now.